test/integration/lighthouse.test.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const assert = require('assertive');
+const co = require('co');
+const getConfig = require('testium-core').getConfig;
+const Semver = require('semver');
+
+const browser = require('../mini-testium-mocha').browser;
+
+const browserName = getConfig().get('browser');
+
+describe('navigation', () => {
+  if (browserName !== 'chrome') {
+    xit('Skipping lighthouse tests. They only work for chrome.');
+    return;
+  }
+
+  if (!Semver.satisfies(process.version, '>=8.0.0')) {
+    xit('Skipping lighthouse tests. They only work for node >=8.0.0.');
```

Comment:
why?

Reply:
and, if true, might as well write the rest of the test file using async/await
we can bump the lint requirement for this project to `groupon/node8` now that 6.x is officially unsupported

Reply:
Removed it since we only support node8+